src/services/MoviesService.js

```diff
@@ -0,0 +1,44 @@
+const fetchURL = "http://localhost:4000/movies";
+
+export const MoviesService = {
```

Comment:
Btw you can use https://redux-toolkit.js.org/rtk-query/overview next time

Reply:
cool, thanks for the advice!